configs/detection_fomo_heavy_model.yaml

```diff
@@ -0,0 +1,42 @@
+# Example configuration for training a predefined fomo heavy detection model
```

Comment:
Add this model into README [here](https://github.com/luxonis/luxonis-train/blob/main/luxonis_train/config/predefined_models/README.md)